### PR TITLE
AUG-297 - Dropdown icon styles

### PR DESCRIPTION
**Pull request by paulkahle**
Includes the following changes to the dropdown icon as described in https://jira.inbcu.com/browse/AUG-297.
1. Reduces the font size to medium
2. Arrow points right when not focused

### CODE REVIEW

File: dist/suprematism.css [hunk 1]
```diff
@@ -3258,13 +3258,16 @@ button, input[type="submit"] {
     z-index: 1000; }
     .dropdown-button:hover .dropdown-label::after, .dropdown-button:focus .dropdown-label::after, .dropdown-button:active .dropdown-label::after {
       font-family: 'Audri-alpha' !important;
-      content: "\e903"; }
+      content: "\e902"; }
```
Comment (craigmichaelmartin) [inline dist/suprematism.css]: Maybe import and use the value from https://github.com/CINBCUniversal/suprematism-font-icons/blob/master/variables.scss#L11

**Comment by paulkahle** [comment]
@craigmichaelmartin I made the change, but there's some issue with the encoding in git.  It shows content: "" in the built css.  This doesn't seem to impact the style in the browser however.

**Comment by craigmichaelmartin** [comment]
Thats odd. Is it correct in the file locally? Are you using the dist version when you see it acting correctly in the browser? If so, :+1:

**Comment by paulkahle** [comment]
Looks like a known sass issue.  Sass is writing the actual unicode character to the css file which is why it displays as "".  https://github.com/sass/sass/issues/659

**Pull request merged**